### PR TITLE
Add Title method to Call interface

### PR DESCRIPTION
**Pull request by osamingo**
Add Title method to Call interface.

### CODE REVIEW

File: stack.go [hunk 1]
```diff
@@ -72,7 +72,7 @@ func funcNameFromFunc(fn *runtime.Func) string {
 	}
 	name := fn.Name()
 	end := strings.LastIndex(name, string(filepath.Separator))
-	return name[end+1 : len(name)]
+	return name[end+1:]
```
Comment (zchee) [inline stack.go:75]: make sense.

File: client_go18_test.go [hunk 2]
```diff
@@ -15,12 +15,12 @@ import (
 var (
 	testTrace = &api.Trace{
 		Frames: []*api.Frame{
-			&api.Frame{
+			{
```
Comment (zchee) [inline client_go18_test.go:18]: Thanks for fix it. more simply :D

BTW, I should remove go
runtime version dependency on test.
Currently uses `errors.New` but that’s depends to line number per go versions. very ugly 😱

**Review by zchee — APPROVED** [review]
seems to LGTM. I’ll merge it.

File: stack.go (hunk 1 above)
Comment (osamingo) [reply stack.go:75]: By `gofmt` 👍

**Pull request merged**